client/blocks/comments/post-comment-list.jsx

```diff
@@ -224,6 +227,22 @@ class PostCommentList extends React.Component {
 		);
 	};
 
+	renderCommentManageLink = () => {
+		const siteId = get( this.props, 'post.site_ID' );
+		const postId = get( this.props, 'post.ID' );
+
+		return (
```

Comment:
Should we return `null` in the case where `siteId` or `postId` are falsey?

Reply:
Yep 👍